ops/scripts/deployer.sh

```diff
@@ -37,5 +37,7 @@ fi
 # build the dump file
 yarn run build:dump
 
-# serve the addrs and the state dump
-exec ./bin/serve_dump.sh
+# service the addresses and dumps
```

Comment:
Oof python2, I wonder if they support python3 with explicit `python3`

Reply:
I tried, didn't work. Would need to install Py3 while building the image.